v2/cmd/wails/internal/commands/dev/dev.go

```diff
@@ -568,6 +601,35 @@ func doWatcherLoop(buildOptions *build.Options, debugBinaryProcess *process.Proc
 					debugBinaryProcess = newBinaryProcess
 				}
 			}
+			if len(changedPaths) != 0 {
+				if assetDir == "" {
+					resp, err := http.Get("http://localhost:34115/wails/assetdir")
+					if err != nil {
+						LogRed("Error during retrieving assetdir: %s", err.Error())
+					} else {
+						content, err := ioutil.ReadAll(resp.Body)
```

Comment:
I think we should be using `io.ReadAll` since Go 1.16 :-)

Reply:
https://tip.golang.org/doc/go1.16#ioutil

Reply:
Oh yes, you are right. I've not yet gotten used to it 😄 .